addons/pinniped/post-deploy/pkg/vars/vars.go

```diff
@@ -8,8 +8,8 @@ import "github.com/vmware-tanzu/tanzu-framework/addons/pinniped/post-deploy/pkg/
 
 var (
 	// PinnipedAPIGroupSuffix is the API group suffix used to talk to Pinniped APIs
-	// E.g., if PinnipedAPIGroupSuffix is set to "tuna.io", then FederationDomain API group would be
-	// "config.supervisor.tuna.io".
+	// E.g., if PinnipedAPIGroupSuffix is the default "pinniped.dev", then FederationDomain API group would be
+	// "config.supervisor.pinniped.dev".
 	PinnipedAPIGroupSuffix = constants.PinnipedDefaultAPIGroupSuffix
```

Comment:
we talked about this over slack but i can't remember what the verdict was...do we still need this? my understanding is no, because it is used to wire through the API group from the CLI, which we don't do anymore

Reply:
Yeah, that's true.

Reply:
Aha, actually yeah this can go away now that you're code is pushed, the custom pinniped clients were using this var.

I'm removing it now.